### PR TITLE
Update defender-vulnerability-management.md

### LINKED ISSUE #12377: Missing reference to Microsoft 365 Defender
Hi,

As this is part of the Microsoft 365 Defender console, it would be useful to state that in the article, mentioning that it is one of the components within the main MS 365 Defender portal.

Cheers
Amjad

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 07d225fa-722b-4608-a695-b935d4b820e1
* Version Independent ID: 07d225fa-722b-4608-a695-b935d4b820e1
* Content: [Microsoft Defender Vulnerability Management](https://learn.microsoft.com/en-us/microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management.md)
* Service: **microsoft-365-security**
* Sub-service: **mdvm**
* GitHub Login: @siosulli
* Microsoft Alias: **siosulli**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the feature available in the portal

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12377

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ ms.date: 05/09/2022
 > [!IMPORTANT]
 > Some information in this article relates to prereleased products/services that might be substantially modified before they are commercially released. Microsoft makes no warranties, express or implied, for the information provided here.
 
+>[!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by siosulli** [comment]
@yogkumgit - the note added refers to Defender for Servers Plan 2 and not Microsoft 365 Defender as mentioned in https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12377.

You can close this issue and I will add a new' applies to' section and note about M365D to the article.

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by siosulli** [comment]
> @yogkumgit - the note added refers to Defender for Servers Plan 2 and not Microsoft 365 Defender as mentioned in #12377.
> 
> You can close this issue and I will add a new' applies to' section and note about M365D to the article.

@yogkumgit  please see this comment. Thanks, Sinead.

**Pull request closed without merging**